tools/js-formatter/README.md

```diff
@@ -0,0 +1,30 @@
+# JS Formatter
+
+This tool validates and formats JavaScript using [`UglifyJS`](https://www.npmjs.com/package/uglifyjs-browser)
```

Comment:
Let's replace the top sentence and links with the new standard pattern

```suggestion
This is a tool to format JS code.

## Factsheet
| **Category**            | **Value**                                  |
|-------------------------|--------------------------------------------|
| **Project Type**        | Tool: Web App                              |
| **Live URL**            | https://js-formatter.holistic-toolbox.com  |
| **Firebase Site Name**  | js-formatter                               |
| **Unique Dependencies** | vue-clipboard, uglifyjs-browser            |
```

Reply:
This change was done in commit 7eaf19f